frontend/src/embedding-sdk-bundle/components/public/dashboard/SdkDashboard.tsx

```diff
@@ -272,6 +273,12 @@ const SdkDashboardInner = ({
     }
   }, [dispatch, dashboardId]);
 
+  useEffect(() => {
+    if (!withSubscriptions) {
+      dispatch(closeSidebarIfSubscriptionsSidebarOpen());
```

Comment:
We shouldn't always close it, since closing the sidebar basically set the sidebar to the `DEFAULT_SIDEBAR` (closed)
https://github.com/metabase/metabase/blob/c356d757fa666f799c059219eca79d0886529bee/frontend/src/metabase/dashboard/reducers-typed.ts#L198

This would be apparent if we have other sidebars open and uncheck `withSubscriptions` it shouldn't close other sidebars.

This action ensures we only close the sidebar when the subscriptions sidebar is open